blockchain/vm/instructions.go

```diff
@@ -584,6 +584,11 @@ func opDifficulty(pc *uint64, evm *EVM, contract *Contract, memory *Memory, stac
 	return nil, nil
 }
 
+func opRandom(pc *uint64, evm *EVM, contract *Contract, memory *Memory, stack *Stack) ([]byte, error) {
+	stack.push(evm.GetHash(evm.BlockNumber.Uint64() - 1).Big())
```

Comment:
Maybe we could add a comment like this

```suggestion
        // evm.BlockNumber.Uint64() is always greater or equal to 1
        // since evm will not run on the genesis block
	stack.push(evm.GetHash(evm.BlockNumber.Uint64() - 1).Big())
```

Reply:
Thank you for the comment~!